spec.yaml

```diff
@@ -553,10 +534,21 @@ definitions:
                       - `read` allows read-only access including download of a copy.
                       - `write` allows create, update, and delete rights on the resource.
                       - `share` allows re-share rights on the resource.
+                    enum: ["read", "write", "share"]
+                requirements:
+                  type: array
+                  items:
+                    type: string
+                    description: |
+                      A list of requirements that the recipient provider MUST fulfill
+                      to access the resource. The share MUST contain at least the `none`
```

Comment:
If `requirements` is optional, shall we instead consider that `none` is implicit? And yet, if it is there, it MUST NOT be empty?

Reply:
And maybe add that (while it's OK to ignore the permissions of a share) the Receiving Server MUST reject a share whose requirements it doesn't understand.